itl/plugins-contrib.d/web.conf

```
@@ -591,6 +591,13 @@ object CheckCommand "ssl_cert" {
 			description = "Ignore the certificate maximum validity"
 			set_if = "$ssl_cert_ignore_maximum_validity$"
                 }
+        	"--long-output" = {
+	                description = "Append the specified comma separated (no spaces) list of attributes to the plugin output on additional lines"
+                	value ="$ssl_cert_long_output$"
```

After I started a change request. Please add a space here - nitpick, I know.


```suggestion
                	value = "$ssl_cert_long_output$"
```